openvdb/openvdb/tools/FastSweeping.h

```diff
@@ -56,6 +56,18 @@ OPENVDB_USE_VERSION_NAMESPACE
 namespace OPENVDB_VERSION_NAME {
 namespace tools {
 
+/// @brief Fast Sweeping update mode. This is useful to determine
+///        narrow-band extension or field extension in one side
+///        of a signed distance field.
+enum class FastSweepingDirection {
+    /// Update all voxels affected by the sweeping algorithm
+    SWEEP_DEFAULT,
```

Comment:
I'm not a fan of DEFAULT.  Sure, it is the  default choice, but if this shows up explicitly in code I'll wonder what DEFAULT means.  Maybe SWEEP_ALL to match SOP token?

Reply:
I agree - SWEEP_FULL or SWEEP_ALL is better. As mentioned in my review comments I don't actually like that this is called a "sweep direction" - I much prefer "sweep domain" (See my comments for an explanation)